gammapy/datasets/map.py

```diff
@@ -475,18 +475,20 @@ def _background_parameters_changed(self):
             self._background_parameters_cached = values
         return changed
 
-    def npred_signal(self, model_name=None):
+    def npred_signal(self, model_name=None, stack=True):
         """Model predicted signal counts.
 
-        If a model name is passed, predicted counts from that component are returned.
-        Else, the total signal counts are returned.
+        If a list of model name is passed, predicted counts from these components are returned.
+        If stack is set to True, a map of the sum of all the predicted counts is returned.
+        If stack is set to False, a map with an additional axis representing the models is returned.
 
         Parameters
         ----------
-        model_name: str
-            Name of  SkyModel for which to compute the npred for.
-            If none, the sum of all components (minus the background model)
-            is returned
+        model_name: list of str
```

Comment:
I guess it should be `model_names` here (with a deprecation warning) to be consistent with our datasets and models naming schemes

Reply:
Done. 
For the deprecation version I put 1.1 but I don't know it is the right one to put.